### ISSUE TITLE
LOHP: Affiliates LP navigation not readable

### ISSUE THREAD

**Issue by nuriapenya** [Automattic/wp-calypso#100495]
## Where
https://wordpress.com/affiliates/

## Problem
The main CTA on the navigation is not readable and do not pass the minimum accessibility standard.

![Image](https://github.com/user-attachments/assets/5c849fed-5520-43a8-958b-66cb600db4f1)

## Solution
I'd propose to change the font color and the border to white.

![Image](https://github.com/user-attachments/assets/6f4e080a-7bc9-442a-b69d-18b03356e69f)

Alternatively, since I see the design was purposely using a black outline navigation, we might want to consider changing the gradient color to something more readable:

![Image](https://github.com/user-attachments/assets/d048801b-722c-4926-9945-d8e4999a9dc2)

@Automattic/dotcom-design, especially brand folks (@Noemish, @jrdnhllr) let me know what would be the desired outcome. I left some notes in this Figma file: WU1iHyqgNsg6mqq0a9GKiF-fi-4150_27864

**Comment by tellthemachines** [comment]
@nuriapenya I'm probably missing context here but what changes do you suggest should be made to the CTA? Should we e.g. change the font color, the background color, both?

**Comment by nuriapenya** [comment]
Thank you @tellthemachines for looking into this! I added a bit more definition to the issue and cc. the dotcom brand team, to double check in couple of solutions.

**Comment by jasmussen** [comment]
I'm also lacking context, but the gradients on this page don't feel as finessed as they could be. A simple solution could be to remove it, even the logo. We already have the WordPress logo context in the corner.

![Image](https://github.com/user-attachments/assets/3d13ff7e-4700-4f51-bb1e-4a922620907f)

Another option would be to be more intentional with the gradients, more a la the plasma on https://automattic.design/blog/. I'd still remove the logomark, however.

**Comment by sfougnier** [comment]
+1 to removing the big WordPress logo. I agree with @jasmussen it's unnecessary and makes the text hard to read.

This gradient seems to work ok, but it would be good to double check without the logo there:
![Image](https://github.com/user-attachments/assets/39ae08bd-e376-434c-973b-e695aca34823)

cc @jrdnhllr @Noemish, if you have other thoughts.

**Comment by tellthemachines** [comment]
Hey folks I just realised that this can be done with just a content update to that specific blog page, it doesn't involve changes to the code at all, so may be best if a designer handles it!

<img width="939" alt="Image" src="https://github.com/user-attachments/assets/23d18a6b-443f-43d4-9c6e-d54422197f14" />

**Comment by chickee0311** [comment]
To expedite the fix, I've made the following updates to the hero on this page: https://wordpress.com/affiliates/

- Removed the logo based on the comments above
- Updated the gradient to closer match the alternative proposal in Figma: https://www.figma.com/design/WU1iHyqgNsg6mqq0a9GKiF/WP.com-Affiliates-LP?node-id=4150-10471&t=GzwwDuFIlK8V8yUe-0
